llvm/lib/Transforms/Vectorize/VPlan.cpp

```diff
@@ -399,8 +400,8 @@ void VPTransformState::packScalarIntoVectorValue(VPValue *Def,
   set(Def, VectorValue, Instance.Part);
 }
 
-BasicBlock *
-VPBasicBlock::createEmptyBasicBlock(VPTransformState::CFGState &CFG) {
+BasicBlock *VPBasicBlock::createEmptyBasicBlock(VPTransformState::CFGState &CFG,
+                                                DomTreeUpdater &DTU) {
```

Comment:
Should DTU reside inside CFGState, now that updating DT is integrated with modifying the CFG?

Reply:
Moved, thanks!